aspnetcore/tutorials/web-api-help-pages-using-swagger.md

```diff
@@ -16,91 +16,61 @@ uid: tutorials/web-api-help-pages-using-swagger
 
 ## NuGet Packages
 
 You can add Swashbuckle with any of the following approaches:
 
 * From the Package Manager Console:
 
-```PowerShell
-Install-Package Swashbuckle.AspNetCore
-   ```
+    ```powershell
+    Install-Package Swashbuckle.AspNetCore
+    ```
 
 * In Visual Studio:
 
-     * Right click your project in Solution Explorer > Manage NuGet Packages
+     * Right-click your project in Solution Explorer > Manage NuGet Packages
      * Enter Swashbuckle.AspNetCore in the search box
      * Set the Package source to nuget.org
      * Tap the Swashbuckle.AspNetCore package and then tap Install
 
 ## Add and configure Swagger to the middleware
 
-Add SwaggerGen to the services collection in the Configure method, and in the ConfigureServices method, enable the middleware for serving generated JSON document and the SwaggerUI.
+Add the Swagger generator to the services collection in the `ConfigureServices` method of *Startup.cs*:
 
-<!-- literal_block {"ids": [], "linenos": false, "xml:space": "preserve", "language": "csharp", "highlight_args": {"hl_lines": [12, 21, 24]}} -->
+[!code-csharp[Main](../tutorials/web-api-help-pages-using-swagger/sample/TodoApi/Startup2.cs?name=snippet_ConfigureServices&highlight=7-10)]
 
-```csharp
-public void ConfigureServices(IServiceCollection services)
-{
-    // Add framework services.
-    services.AddMvc();
-
-    services.AddLogging();
-
-    // Add our repository type
-    services.AddSingleton<ITodoRepository, TodoRepository>();
+In the `Configure` method of *Startup.cs*, enable the middleware for serving the generated JSON document and the SwaggerUI:
 
-    // Register the Swagger generator, defining one or more Swagger documents
-    services.AddSwaggerGen(c =>
-    {
-        c.SwaggerDoc("v1", new Info { Title = "My API", Version = "v1" });
-    });
-}
-
-// This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
-public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerFactory loggerFactory)
-{
-    app.UseMvcWithDefaultRoute();
+[!code-csharp[Main](../tutorials/web-api-help-pages-using-swagger/sample/TodoApi/Startup2.cs?name=snippet_Configure&highlight=4,7-10)]
 
-    // Enable middleware to serve generated Swagger as a JSON endpoint.
-    app.UseSwagger();
-
-    // Enable middleware to serve swagger-ui (HTML, JS, CSS etc.), specifying the Swagger JSON endpoint.
-    app.UseSwaggerUI(c =>
-    {
-        c.SwaggerEndpoint("/swagger/v1/swagger.json", "My API V1");
-    });
-}
-```
-
-In Visual Studio, press ^F5 to launch the app and navigate to `http://localhost:<random_port>/swagger/v1/swagger.json` to see the document generated that describes the endpoints.
+Press *F5* in Visual Studio to launch the app. Navigate to `http://localhost:<random_port>/swagger/v1/swagger.json` to see the generated document that describes the endpoints.
```

Comment:
~Press *F5* in Visual Studio to launch the app~
This is not a beginning tutorial - let's keep it as tool agnostic as possible. 
Launch the app and navigate to  `http://localhost:<port>/swagger/v1/swagger.json`